docs/DeveloperGuide.adoc

```diff
@@ -8,16 +8,16 @@
 
 *Prerequisites*
 
-* JDK 8
-* IntelliJ IDE
+* JDK 11
+* IntelliJ IDEA
```

Comment:
AB-1, AB-3 and AB-4 all use "IntelliJ IDE", so I'm not sure if it makes sense to make AB-2 inconsistent with the others, especially since this PR is just supposed to do maintenance. Shall we remove this change?

Reply:
Let's move postpone this change to the documentation overhaul later on. Strictly speaking `IntelliJ IDEA` is the name of the IDE but that's a discussion that we can have in a separate PR.

Reply:
Yes, it is important to strictly separate commits between those that are simply performing maintenance (refactoring etc.) and enhancements.